boot.py

```diff
@@ -22,10 +24,13 @@
 from .plugin.core.protocol import Error
 from .plugin.core.protocol import Location
 from .plugin.core.protocol import LocationLink
+from .plugin.core.registry import LspCollapseTreeItemCommand
+from .plugin.core.registry import LspExpandTreeItemCommand
 from .plugin.core.registry import LspNextDiagnosticCommand
 from .plugin.core.registry import LspPrevDiagnosticCommand
 from .plugin.core.registry import LspRecheckSessionsCommand
 from .plugin.core.registry import LspRestartServerCommand
+from .plugin.core.registry import LspWindowCommand
```

Comment:
I wonder if we should move LspOpenLocationCommand somewhere else (core/registry.py?), and just import it in this file, so that this import line can be removed. Currently this `lsp_window` command would be exposed to users, but it has no real use by itself.

Reply:
Since there were no comments, and I didn't see a reason to have this single command being defined in `boot.py`, I moved it into `core/registry.py`. If there are any objections, I could revert the commit.